File: OMSWeb/frontend/packages/oms-track-monitor/src/TrackObjects/point/types/Point.ts
import { CanBeFocused } from 'src/MapObjects/focus/types/CanBeFocused'

type Point = {
	id: number
	logicalId: string
	physicalId: string
	x: number
	y: number
	z?: number

	homeId?: number
} & CanBeFocused

export { Point }

File: OMSWeb/Models/Tracks/Point.cs
namespace OMSWeb.Models.Tracks
{
    public class Point
    {
        public int Id { get; set; }
        public string PhysicalId { get; set; }
        public string LogicalId { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int? HomeId { get; set; }
    }
}